client/src/components/Container.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components/macro";
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  /* grid-template-rows: 3fr 5fr;
+  justify-content: center; */
```

Comment:
I would avoid unused code to keep the code clean

Reply:
Thanks, didn´t see it.